### ISSUE TITLE
Hide accessibility features on smaller displays (accessibility features)

### ISSUE THREAD

**Issue by WM-Loose** [joomla/joomla-cms#39860]
### Steps to reproduce the issue
1. Enable accessibility features an look on small display
![1](https://user-images.githubusercontent.com/99647423/218695664-42d0c7c9-f6e7-45c0-9ece-1b115657e95c.png)

* What version of Joomla were you using?
4.2.27

### Additional comments
Is it possible to choose to hide the accessibility features on medium and small displays?
Didn't find a setting in the plugin or is this not taken into account at all?

It is rather annoying on small displays such as smartphones and cannot be hidden. 
Was this option forgotten in the plugin?

**Comment by sandewt** [comment]
Related to:  #39161 and #39615<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39860">issues.joomla.org/tracker/joomla-cms/39860</a>.</sub>

**Comment by sandewt** [comment]
> Is it possible to choose to hide the accessibility features on medium and small displays?
Didn't find a setting in the plugin or is this not taken into account at all?

This requires an adjustment in the .js code, as far as I can tell right now.

> It is rather annoying on small displays such as smartphones and cannot be hidden.
Was this option forgotten in the plugin?

This is possible to modify some code in the plugin. Here's a simple setup. But that is of no use to you now.

```
public function onAfterRender()
{
	// Check if mobile
	if (Factory::getApplication()->client->mobile) {
		return;
	}
}
```

**Comment by ChristineWk** [comment]
@sandewt 

The specified issues are open, but concern other topics, I think ...
But Thanks for your interest!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39860">issues.joomla.org/tracker/joomla-cms/39860</a>.</sub>

**Comment by Quy** [comment]
@brianteeman For #43143?

**Comment by brianteeman** [comment]
Not something I would be prepared to add to the plugin as I consider it bad practice. However the new version will (not implemented yet) allow custom css so you could do it if you really want to.
